webui/react/src/components/UPlotChart.tsx

```diff
@@ -12,17 +13,27 @@ interface Props {
   options?: Options;
 }
 
+const SCROLL_THROTTLE_TIME = 500;
+
 const UPlotChart: React.FC<Props> = ({ data, options }: Props) => {
   const [ chart, setChart ] = useState<uPlot>();
   const chartDivRef = useRef<HTMLDivElement>(null);
 
-  // Chart setup.
+  /*
+   * Chart setup.
+   */
   useEffect(() => {
     if (!chartDivRef.current || !options) return;
 
-    if (!options.width) options.width = chartDivRef.current.offsetWidth;
+    const optionsExtended = uPlot.assign(
+      {
+        cursor: { drag: { dist: 10, uni: 100, x: true, y: true } },
```

Comment:
**suggestion:** I've played around with this section a little bit. I recommend `dist` to be a little bit smaller of `5` and `uni` to be `null`. With `dist` just a slight positive value should be a good indication they want to drag and zoom. The `uni` of `null` will ensure all drag and zoom will be both x and y and avoid any uni-direction zoom which made no sense to me.